tests/lib/config/config-file.js

```diff
@@ -349,6 +365,32 @@ describe("ConfigFile", function() {
             assert.deepEqual(config, updatedConfig.eslintConfig);
         });
 
+        it("should load fresh information from a .eslintrc.js file", function() {
+            var initialConfig = {
```

Comment:
Won't this leave a temp file hanging around? I don't think you have to overwrite a file to test this functionality.